vaadin-item.html

```diff
@@ -25,8 +40,9 @@
   <template>
     <style>
       :host {
-        display: block;
+        display: inline-block;
         cursor: pointer;
+        flex-shrink: 0;
```

Comment:
- Use `display: flex;` and `align-items: center;`
  - I’m not 100% sure if these are already good to have, but I’m expecting us to use flexbox once we add the planned slots (icon/prefix, label/description and badge/action)
- Remove flex-shrink (only needed for `vaadin-tab`).
- Remove cursor, only needed when used inside a list element.

Reply:
Below: remove the disabled state related styles.